test/parameter_test.cc

```diff
@@ -35,103 +35,104 @@ using namespace keyvaluestore;
 
 int logTime = -1;
 
-namespace acap_runtime {
-namespace parameter_test {
-
-void LogMemory()
+namespace acap_runtime
 {
-  time_t now = time(NULL);
-  struct tm *nTime = localtime(&now);
-  int nowTime = nTime->tm_hour;
-  if (nowTime != logTime)
+  namespace parameter_test
```

Comment:
Starting with C++17, you can write this as `namespace acap_runtime::parameter_test`